comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -255,9 +258,34 @@ func (ia *inventoryagent) initData() {
 
 	ia.data["feature_dynamic_instrumentation_enabled"] = getBoolSysProbe("dynamic_instrumentation.enabled")
 
+	// ECS Fargate
+	ia.fetchECSFargateAgentMetadata()
+
 	ia.refreshMetadata()
 }
 
+func (ia *inventoryagent) fetchECSFargateAgentMetadata() {
+	// ECS Fargate agent task ARN
+	if !env.IsECSFargate() {
+		return
+	}
+	client, err := ecsmeta.V2()
+	if err != nil {
+		ia.log.Debugf("error while initializing ECS metadata V2 client: %s", err)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Shouldn't it be at least `Info` ?

Reply:
Set it to `Warn` 👍